src/type/restaurant.ts

```diff
@@ -0,0 +1,14 @@
+type Category = "한식" | "중식" | "일식" | "아시안" | "양식" | "기타" | "전체";
+
+type Distance = "5" | "10" | "15" | "20" | "30";
+
+interface Restaurant {
+  name: string;
+  category: Category;
+  distance: Distance;
+
+  description: string;
+  link: string;
```

Comment:
```suggestion
  description?: string;
  link?: string;
```
확인해보니 필수값이 아니더라구요~
이렇게 표현해보면 어떨까요?

Reply:
필수값이 아닐 때 저렇게 ?를 사용할 수 있는 지 몰랐어요! 감사합니다😃😃